core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -434,6 +437,44 @@ export class NavigationComponent {
   onClickGetAllVersions() {
     this.workflowVersionService.clickDisplayWorkflowVersions();
   }
+
+  private handleVersionDisplay(): void {
+    this.workflowVersionService
+      .getDisplayParticularVersionStream()
+      .pipe(untilDestroyed(this))
+      .subscribe(displayVersionFlag => {
+        this.particularVersionDate =
+          this.workflowActionService.getWorkflowMetadata().creationTime === undefined
+            ? ""
+            : "" +
+              this.datePipe.transform(
+                this.workflowActionService.getWorkflowMetadata().creationTime,
+                "MM/dd/yyyy HH:mm:ss zzz",
+                Intl.DateTimeFormat().resolvedOptions().timeZone,
+                "en"
```

Comment:
I am not sure whether we have another case during the same logic (for example we are displaying workflow last modified time originally). I think your version is better, so two questions:
1. are we handling time zones correctly? aka we store in db the UTC time, but display localized time (based on user's browser).
2. can this logic be reused for other places we display timestamps? (in user dashboard and on version timestamps, etc). I wonder if it is a good idea to have a time util file some where.

Reply:
I actually am using the same logic used for storing and displaying the workflow. I like the idea of time util maybe we add it later